lib/recurly/token.js

```diff
@@ -186,4 +188,19 @@ function token (customerData, bus, done) {
     }
     done(null, res);
   }
+
+  function enrichInputs (recurly, inputs, results) {
+    if (results.length === 0) return;
+
+    inputs.risk = [];
+
+    results.forEach(result => {
+      if (result.processor === 'braintree_blue') {
+        inputs.proactive = recurly.config.risk.threeDSecure.proactive;
+        inputs.proactive.params = result;
+      } else {
+        inputs.risk.push(result);
+      }
+    });
+  }
```

Comment:
Let's allow the results to append to `inputs.risk` as normal. Should be just as effective.